FlowCrypt/src/main/res/values/strings.xml

```diff
@@ -379,8 +379,7 @@
     <string name="your_email_address">Your email address</string>
     <string name="pending">Pending&#8230;</string>
     <string name="mark_unread">Mark unread</string>
-    <string name="user_not_verified">User not verified</string>
-    <string name="user_not_registered_not_verified">User not registered and not verified</string>
+    <string name="user_not_registered_or_verified">User is not registered or not verified</string>
```

Comment:
@tomholub Please look at this line. If we are going to drop `isRegistered` should we show `User is not registered or not verified` explanation? Maybe it should be just `User not verified`?

Reply:
Correct, just "User not verified successfully"

Reply:
Got it. I'm going to fix it right now